charts/tidb-operator/templates/controller-manager-rbac.yaml

```diff
@@ -38,7 +38,7 @@ rules:
   verbs: ["get", "list", "watch", "create", "update", "delete"]
 - apiGroups: [""]
   resources: ["secrets"]
-  verbs: ["create", "get", "list", "watch"]
+  verbs: ["create", "update", "get", "list", "watch"]
```

Comment:
does this should be consistent with https://github.com/pingcap/tidb-operator/pull/1954/files#diff-ecdc286a1f57077063e4a16696198f7fR138?

Reply:
updated.